packages/engine-server/src/DendronEngineV3.ts

```diff
@@ -1022,6 +1166,45 @@ export class DendronEngineV3 extends EngineV3Base implements DEngine {
     };
   }
 
+  private createRenderedCache(
```

Comment:
I'm not opposed to leaving it since you've already added it, but I think it makes sense to get rid of the preview caching logic.  I'm not convinced it's actually causing any perceivable perf gains and it makes the render logic more complex without a good reason. 

From profiling I did in the past, if we can't render a note in <50 ms then we have a bug in our rendering logic and that's where we should make a fix.

Reply:
I'm not familiar enough to make a decision here, but I would definitely lean towards simplicity if we can. Any thoughts @kevinslin

Reply:
i would leave it in for now and remove it in a subsequent pr if we don't need it

Reply:
just adding to the conversation that I have triaged some issues with preview where some nested note references were not being rendered due to caching issues.